tests/foreman/ui/test_dashboard.py

```diff
@@ -0,0 +1,485 @@
+"""Test module for Dashboard UI"""
+from robottelo.decorators import stubbed, tier3
+from robottelo.test import UITestCase
+
+
+class DashboardTestCase(UITestCase):
+    """Tests for Dashboard UI"""
+
+    @stubbed()
+    @tier3
+    def test_positive_save(self):
+        """@Test: Save the Dashboard UI
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Dashboard
+        2.Try to remove some widgets
+        3.Select the Manage Dropdown box
+        4.Save the Dashboard
+
+        @Assert: Dashboard is saved successfully
+        and the removed widgets does not appear.
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_reset(self):
+        """@Test: Reset the Dashboard to default UI
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Dashboard
+        2.Try to remove some widgets
+        3.Select the Manage Dropdown box
+        4.Save the Dashboard
+        5.Dashboard Widgets are saved successfully
+        6.Click Reset to default
+
+        @Assert: Widget positions successfully saved.
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_add_widgets(self):
+        """@Test: Add Widgets to Dashboard UI
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Dashboard
+        2.Select Manage Dropdown box
+        3.Add Widgets
+
+        @Assert: User is able to add widgets.
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_search_bookmark(self):
+        """@Test: Bookmark the search filter in Dashboard UI
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Dashboard
+        2.Add a filter to search box (eg. environment)
+        3.Bookmark the search filter
+
+        @Assert: User is able to list the Bookmark
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_host_configuration_status(self):
+        """@Test: Review the Host Configuration Status widget
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Dashboard
+        2.Focus on the Host Configuration Status
```

Comment:
I already commented to change this to 'Review the Host Configuration Status widget'.